webapp/src/main/java/com/box/l10n/mojito/entity/PullRunTextUnitVariant.java

```diff
@@ -22,12 +22,7 @@
  * @author garion
  */
 @Entity
-@Table(
-        name = "pull_run_text_unit_variant",
-        indexes = {
-                @Index(name = "UK__PULL_RUN_TEXT_UNIT_VARIANT__EVA_ID__TM_TUV_ID", columnList = "pull_run_asset_id, tm_text_unit_variant_id", unique = true)
```

Comment:
remove the constraint since we can have duplicated tm text unit variant id. 

If we want to put it back we need to add a `locale_id` column

Reply:
Would we not need locale_id added anyways after removing this constraint?
Otherwise - we'd have exactly the same TUVs duplicated - and we wouldn't be able to discern any useful information from the duplicates - if we don't specifically know what locales they were intended for.

Just duplicating the data without a locale_id will bloat the DB.

Reply:
I agree, I feel like having the `locale_id` makes more sense to cover inheritance. Do we want to denormalize the `tm_text_unit_id` also, we'd have a table pretty similar to the current variants then